From e76fe7ee2ad1143bc0dd532553f612e634f89944 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marek=20Marczykowski-G=C3=B3recki?=
 <marmarek@invisiblethingslab.com>
Date: Sat, 21 Dec 2019 10:04:13 +0100
Subject: [PATCH] Workaround for tornado 6

tornado.stack_context is no longer available in tornado 6 library.
Comment out the import as a workaround. The actual thing seems to not be
used in our specific cases.
---
 salt/client/mixins.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/salt/client/mixins.py b/salt/client/mixins.py
index 480145e..6798c07 100644
--- a/salt/client/mixins.py
+++ b/salt/client/mixins.py
@@ -33,7 +33,7 @@ import salt.log.setup
 from salt.ext import six
 
 # Import 3rd-party libs
-import tornado.stack_context
+#import tornado.stack_context
 
 log = logging.getLogger(__name__)
 
-- 
2.21.0

